tools/attrd_updater.c

```diff
@@ -175,6 +195,12 @@ static GOptionEntry addl_entries[] = {
       "If this creates a new attribute, never write the attribute to CIB",
       NULL },
 
+    { "wait", 'W', 0, G_OPTION_ARG_CALLBACK, wait_cb,
+      "Wait until the attribute change has propagated to the given point\n"
+      INDENT "before returning.  Values are 'no', 'local', 'cluster'.  Default\n"
+      INDENT "is 'no' (return immediately).",
```

Comment:
I'd be more explicit about what is promised, i.e. local means that a local query will return the requested value (unless changed by a later request)

Also bump feature set

Reply:
Technically 'no' does wait for the daemon to ack the request

Reply:
Updated text for 'local', mention that 'no' does wait for something, and remove mention of 'cluster' for now.